gpkit/__init__.py

```diff
@@ -95,6 +96,50 @@ def __call__(self, arg):
 
 enable_units()
 
+VECTORIZATION = ()
+MODELS = []
+MODELNUMS = []
+MODELNUM_LOOKUP = defaultdict(int)
+
+
+def begin_variable_naming(model):
+    global MODELS, MODELNUMS
+    MODELS.append(model)
+    num = MODELNUM_LOOKUP[model]
+    MODELNUMS.append(num)
+    MODELNUM_LOOKUP[model] += 1
+    return num, (tuple(MODELS), tuple(MODELNUMS))
+
+
+def end_variable_naming():
+    global MODELS, MODELNUMS
+    MODELS = MODELS[:-1]
+    MODELNUMS = MODELNUMS[:-1]
+
+
+class NamedVariables(object):
+    def __init__(self, model):
+        self.model = model
+
+    def __enter__(self):
+        begin_variable_naming(self.model)
+
+    def __exit__(self, type_, val, traceback):
+        end_variable_naming()
```

Comment:
this class appears unused in the code base except for one occurrence in `t_model`. Do we still need it?